crashtracker/src/stacktrace.rs

```diff
@@ -33,16 +34,54 @@ pub struct StackFrame {
     pub names: Option<Vec<StackFrameNames>>,
     #[serde(skip_serializing_if = "Option::is_none")]
     #[serde(default)]
+    pub normalized_ip: Option<NormalizedAddress>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    #[serde(default)]
     pub sp: Option<String>,
     #[serde(skip_serializing_if = "Option::is_none")]
     #[serde(default)]
     pub symbol_address: Option<String>,
 }
 
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+pub enum NormalizedAddressMeta {
+    Apk(PathBuf),
+    Elf {
+        path: PathBuf,
+        build_id: Option<Vec<u8>>,
+    },
+    Unknown,
+    Unexpected(String),
+}
+
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+pub struct NormalizedAddress {
+    pub file_offset: u64,
+    pub meta: NormalizedAddressMeta,
+}
+
 #[cfg(unix)]
 mod unix {
     use super::*;
-    use blazesym::symbolize::{Input, Source, Sym, Symbolized, Symbolizer};
+    use blazesym::{
+        normalize::{Normalizer, UserMeta},
+        symbolize::{Input, Source, Sym, Symbolized, Symbolizer},
+        Pid,
+    };
+
+    impl From<&UserMeta> for NormalizedAddressMeta {
+        fn from(value: &UserMeta) -> Self {
+            match value {
+                UserMeta::Apk(a) => Self::Apk(a.path.clone()),
+                UserMeta::Elf(e) => Self::Elf {
+                    path: e.path.clone(),
+                    build_id: e.build_id.clone(),
```

Comment:
Does this handle go build id, which can be in a different part of the metadata than the GNU build id? Or is this irrelevant because we don't currently use libdatadog in dd-trace-go?

Reply:
I'd guess the second, but Felix/Nick may have thoughts.  If they want it, I'll add it in a followup PR